.eslintrc.json

```diff
@@ -12,7 +12,6 @@
     "plugin:react/recommended",
     "plugin:jsx-a11y/recommended",
     "plugin:react-hooks/recommended",
-    "plugin:jest/recommended",
```

Comment:
I assume `jest` was removed from VS Code because it was running tests with every change right?

We actually turn off auto run by adding the key-value pair to our `settings.json` in VS Code:

`"jest.autoRun": "off"`